### PR TITLE
chore: Add reddit and twitter to release, move website event to pre-release

### PR DESCRIPTION
**Pull request by ebebbington**
## Summary

* Move website event to pre release
* Add automation for sending tweet and make reddit post

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -15,6 +15,25 @@ jobs:
...
+  send-twitter-tweet:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: ethomson/send-tweet-action@v1
+        with:
+          status: "New Dmm Version Released! https://github.com/drashland/drash/releases/latest"
+          consumer-key: ${{ secrets.TWITTER_CONSUMER_API_KEY }}
+          consumer-secret: ${{ secrets.TWITTER_CONSUMER_API_SECRET }}
+          access-token: ${{ secrets.TWITTER_ACCESS_TOKEN }}
+          access-token-secret: ${{ secrets.TWITTER_ACCESS_TOKEN_SECRET }}
```
Comment (Guergeiro) [inline .github/workflows/release.yml:39]: I would join everything and call it a *Socials*. Booting a ubuntu VM for each one seems overkill 😃 
```yml
socials:
    runs-on: ubuntu-latest
    steps:
        - uses: bluwy/release-for-reddit-action@v1
          name: Publish to r/deno
          with:
            username: ${{ secrets.REDDIT_USERNAME }}
            password: ${{ secrets.REDDIT_PASSWORD }}
            app-id: ${{ secrets.REDDIT_APP_ID }}
            app-secret: ${{ secrets.REDDIT_APP_SECRET }}
            subreddit: deno
            comment: ${{ github.event.release.body }}
        - uses: ethomson/send-tweet-action@v1
          name: Publish to twitter
          with:
            status: "New Dmm Version Released! https://github.com/drashland/drash/releases/latest"
            consumer-key: ${{ secrets.TWITTER_CONSUMER_API_KEY }}
            consumer-secret: ${{ secrets.TWITTER_CONSUMER_API_SECRET }}
            access-token: ${{ secrets.TWITTER_ACCESS_TOKEN }}
            access-token-secret: ${{ secrets.TWITTER_ACCESS_TOKEN_SECRET }}
```

**Review by Guergeiro — DISMISSED** [review]
I haven't tested the actions, but everything seems to be okay.

File: .github/workflows/release.yml (hunk 1 above)
Comment (ebebbington) [reply .github/workflows/release.yml:39]: Yeah great idea! :D I'll do that for all of the repos too 👍

Comment (crookse) [reply .github/workflows/release.yml:39]: that's a really good idea

Comment (ebebbington) [reply .github/workflows/release.yml]: https://github.com/drashland/dmm/pull/85/commits/b59fc30ff008d68b5db127257475347c793e9341

**Comment by ebebbington** [comment]
deffo ready for a review

**Comment by ebebbington** [comment]
@Guergeiro you good to review again?

**Review by crookse — APPROVED** [review]

**Review by Guergeiro — APPROVED** [review]
Seems good to me!

**Pull request merged**